### DISCUSSION TITLE
Does this conflict with (old) New Xkit 7?

### DISCUSSION THREAD

**Discussion by arcanetrivia** [AprilSylph/XKit-Rewritten#161] [Help Desk] [answered]
Can this extension and New Xkit be run side by side, or should only one be installed/enabled in the browser at any one time? (I think the answer is likely to be "only one at a time" because they are trying to work on the same pages, but I couldn't find any explicit guidance.)

**Comment by AprilSylph** [comment, marked as answer]
They can be run simultaneously; scripts which attempt to do the same things as any XKit 7 extensions attempt to override them (like with Quick Reblog and Timestamps) or not touch each other (like PostBlock or NotificationBlock). If any problems arise from running this addon on top of XKit 7, please open an issue describing the unwanted behaviour.
